### ISSUE TITLE
Issue with Code Execution: Parsing Error

### ISSUE THREAD

**Issue by sumit-kumar-9297** [mntlra/DEXTER#1]
Hello,

I trust this message finds you well.

I encountered an issue while attempting to execute the provided code. Upon execution, I generated a log file named "dexter.log," which indicates that although there are a total of 6189 documents, they are not being parsed for extracting relations as expected.

Could you kindly guide me in resolving this issue? I have attached the aforementioned log file for your reference.

Thank you very much for your attention to this matter.

Best regards,
Sumit Kumar
[dexter.log](https://github.com/mntlra/DEXTER/files/15258569/dexter.log)

**Comment by mntlra** [comment]
Dear Kumar,

Thank you for reaching out.

I tested the code and did not encounter the issue you are experiencing. In particular,  I cloned the repository, set a new conda environment, and ran the instruction provided in the README file and had no problem. I made some small adjustments to the code and the requirements list so I suggest you pull the repository again, check if your environment is set correctly, and follow the updated instructions to run the DEXTER_data. Make sure you have correctly installed the "en-sci-sm" model from Scispacy (https://allenai.github.io/scispacy/). 

For reference, I attach my log file (file: "dexter_output.log") and the configuration file of my conda environment (file: "environment.txt"). If you want to build an identical conda environmental starting from the configuration file:

1. Change the extension of the file from ".yml" to ".txt" (I had to save it as .txt to upload it here)
2. Create a conda environment following the instructions at: [creating an environment from an environment yml file](https://conda.io/projects/conda/en/latest/user-guide/tasks/manage-environments.html#creating-an-environment-from-an-environment-yml-file)

Let me know if this works.

Kind Regards,
Laura Menotti

[dexter_output.log](https://github.com/mntlra/DEXTER/files/15273728/dexter_output.log)
[environment.txt](https://github.com/mntlra/DEXTER/files/15273748/environment.txt)
